src/commands/graph/graph-init.js

```diff
@@ -0,0 +1,114 @@
+// @ts-check
+const { Buffer } = require('buffer')
+
+const { v4: uuidv4 } = require('uuid')
+
+const { OneGraphCliClient, ensureCLISession } = require('../../lib/one-graph/cli-client')
+const { NETLIFYDEVERR, chalk, error, log } = require('../../utils')
+
+const { ensureAppForSite, executeCreateApiTokenMutation } = OneGraphCliClient
+
+/**
+ * Creates the `netlify graph:init` command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ * @returns
+ */
+const graphInit = async (options, command) => {
+  const { api, site, state } = command.netlify
+  const siteId = site.id
+
+  if (!site.id) {
```

Comment:
[dust] You're creating `siteId` but still using `site.id` here. Perhaps we could get rid of `siteId` or use it consistently instead of `site.id`?